source/globalCommands.py

```diff
@@ -4919,9 +4919,10 @@ def script_toggleApplicationsMute(self, gesture: "inputCore.InputGesture") -> No
 
 	@script(
 		# Translators: Describes a command.
-		description=_("""Mute or unmute the speech coming from the remote computer"""),
+		description=_("Mute or unmute the speech coming from the remote computer"),
```

Comment:
could we change these strings to be pgettext?
Or create an issue to convert these separately?

```suggestion
		description=pgettext("remote", "Mute or unmute the speech coming from the remote computer"),
```

Reply:
I'd like to convert Remote to using pgettext across the board, so I'll open a new issue for it.